src/main/java/seedu/address/logic/commands/AssignCommand.java

```diff
@@ -54,18 +58,18 @@ public AssignCommand(Index taskIndex, Index personIndex) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        List<Person> shownPersonList = model.getFilteredPersonList();
-        List<Task> shownTaskList = model.getFilteredTaskList();
+        List<Person> filteredPersonList = model.getFilteredPersonList();
```

Comment:
If you are concerned about the naming, you may want to look at `EditCommand` which uses the naming `lastShownList` (I don't think it matters, should be fine either way)